commons/Utils.py

```diff
@@ -1,5 +1,5 @@
 import re
 
 class Utils():
-    
-    COMMA_DELIMITER = re.compile(''',(?=(?:[^'"]|'[^']*'|"[^"]*")*$)''')
```

Comment:
why do we need to make this small changes?

Reply:
I made some changes to the original one in scala to run in python (because the languages parse regex in a different way), and among those changes I, mistakenly, made the new one not match commas within single quotations too. But this raised some problems when processing the stackoverflow survey. I just removed that.